packages/table/src/table.tsx

```diff
@@ -41,6 +42,14 @@ import { TableBody } from "./tableBody";
 
 export interface ITableProps extends IProps, IRowHeights, IColumnWidths {
     /**
+     * If `true`, there will be a single "focused" cell at all times,
+     * which can be used to interact with the table as though it is a
+     * spreadsheet.
```

Comment:
...true. 

Suggestions here? Currently the combination of the two does inconsistent things. None of the callbacks work unless it's turned on. Should we just ignore the controlled mode version if it's false? Actually that seems most consistent I'm just going to do that.